solutions/devsprint-adriano-song-1/RealEstateApp/Screens/Account/AccountViewController.swift

```diff
@@ -11,7 +11,11 @@ class AccountViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        self.view.backgroundColor = .white
+        navigationItem.title = "Conta"
+        //self.view.backgroundColor = .white
+    }
+    
+    override func loadView() {
+        self.view = LoginView()
```

Comment:
you don't need to create a subview called LoginView and embedded on AccountViewController, but that's okey this one